Gaijinizer patch/Map389.txt

```diff
@@ -25,96 +27,116 @@
 早く教えちゃったほうが身のためだと思うなぁー？
 > CONTEXT: Map389/events/3/pages/0/18/Dialogue/enemy29
 > CONTEXT: Map389/events/11/pages/0/18/Dialogue/enemy29
-You Faeries are like butterflies, and just
-getting your wings pinched like this will weaken
-them, right?
+You pixies are just like butterflies, I just
+have to pinch your wings like this and you go
+limp.
 
-I think it's better for you to tell me sooner
-rather than later, don't you?#DL
+I think you should hurry up and tell me what you
+know, it'll be a lot safer for you that way.#Ron
 > END STRING
 
 > BEGIN STRING
 し、知らない……
 
 に……ニンゲンになんて……会ってない……
-> CONTEXT: Map389/events/3/pages/0/23/Dialogue/enemy27 < UNTRANSLATED
-> CONTEXT: Map389/events/11/pages/0/23/Dialogue/enemy27 < UNTRANSLATED
+> CONTEXT: Map389/events/3/pages/0/23/Dialogue/enemy27
+> CONTEXT: Map389/events/11/pages/0/23/Dialogue/enemy27
+I-I don't know anything...
 
+I-I haven't met any humans...#Ron
 > END STRING
 
 > BEGIN STRING
 ウソはよくないなぁ……
 
 ボクはそこらの魔獣よりかはハナが利くんだ
 キミから、微かにニンゲンの匂いがするんだよ
-> CONTEXT: Map389/events/3/pages/0/28/Dialogue/enemy29 < UNTRANSLATED
-> CONTEXT: Map389/events/11/pages/0/28/Dialogue/enemy29 < UNTRANSLATED
-
+> CONTEXT: Map389/events/3/pages/0/28/Dialogue/enemy29
+> CONTEXT: Map389/events/11/pages/0/28/Dialogue/enemy29
+It's not good to lie.
+Unlike that beast you've got over there, I have
+quite the keen nose, and you've got the faint
+stink of a human about you.#Ron
 > END STRING
 
 > BEGIN STRING
 大丈夫、そのニンゲンに危害は加えたりしないよ
 
 ちょっと遊んでほしいなって思ってるだけ♪
-> CONTEXT: Map389/events/3/pages/0/33/Dialogue/enemy29 < UNTRANSLATED
-> CONTEXT: Map389/events/11/pages/0/33/Dialogue/enemy29 < UNTRANSLATED
+> CONTEXT: Map389/events/3/pages/0/33/Dialogue/enemy29
+> CONTEXT: Map389/events/11/pages/0/33/Dialogue/enemy29
+Don't worry, I'm not going to hurt the human.
 
+I just wanted to have a little fun.#Ron
 > END STRING
 
 > BEGIN STRING
 キミにだって、別に乱暴しようとか思ってないよ
 
 ただ、あんまりボクに協力的じゃなかったら
 すこし怖い目に遭っちゃうかもしれないよー？
-> CONTEXT: Map389/events/3/pages/0/37/Dialogue/enemy29 < UNTRANSLATED
-> CONTEXT: Map389/events/11/pages/0/37/Dialogue/enemy29 < UNTRANSLATED
-
+> CONTEXT: Map389/events/3/pages/0/37/Dialogue/enemy29
+> CONTEXT: Map389/events/11/pages/0/37/Dialogue/enemy29
+I don't really want to get violent with you. But
+if you're not going to be cooperative then I
+might have to give you a little scare.#Ron
 > END STRING
```

Comment:
This might not directly be in the text, but I think this small change will make it look a bit better:
```
...I don't really want to get violent with you.

But if you're not going to be cooperative, then
I might have to give you a little scare.
```